.gitlab-ci.yml

```diff
@@ -728,9 +728,9 @@ zombienet-tests-parachains-disputes:
 
 zombienet-test-parachains-upgrade-smoke-test:
   stage:                           stage3
-  image:                           "${ZOMBIENET_IMAGE}"
   <<:                              *kubernetes-env
   <<:                              *zombienet-refs
+  image:                           "docker.io/paritytech/zombienet:v1.2.45"
```

Comment:
can we instead update the env variable `ZOMBIENET_IMAGE`?

Reply:
Yes, that will update the image for all the tests. I can do that in a followup pr.
Thx!